applications/nrf_desktop/doc/led_state.rst

```diff
@@ -1,6 +1,54 @@
+  * :cpp:enumerator:`LED_SYSTEM_STATE_CHARGING` - the device is being charged,
+  * :cpp:enumerator:`LED_SYSTEM_STATE_ERROR` - fatal application error occurred (a module reported error state or battery state error was reported).
+
+* :cpp:enumerator:`LED_ID_PEER_STATE` - shows Bluetooth peer state. The Bluetooth peer can be in one of the following states:
+
+  * :cpp:enumerator:`LED_PEER_STATE_DISCONNECTED` - the Bluetooth peer is disconnected,
+  * :cpp:enumerator:`LED_PEER_STATE_CONNECTED` - the Bluetooth peer is connected,
+  * :cpp:enumerator:`LED_PEER_STATE_CONFIRM_SELECT` - the Bluetooth peer is being selected (the device is waiting for confirmation),
+  * :cpp:enumerator:`LED_PEER_STATE_CONFIRM_ERASE` - the device is waiting for user confirmation to erase peers (for Bluetooth central) or start erase advertising (for Bluetooth peripheral),
+  * :cpp:enumerator:`LED_PEER_STATE_ERASE_ADV` - the device is advertising for peer erase.
+
+Complete description of peer management is available in :ref:`ble_bond`.
+
+Module Events
+*************
+
++------------------------+------------------------------+---------------+----------------+-------------+
+| Source Module          | Input Event                  | This Module   | Output Event   | Sink Module |
++========================+==============================+===============+================+=============+
+| :ref:`ble_bond`        | ``ble_peer_operation_event`` | ``led_state`` | ``led_event``  | :ref:`leds` |
++------------------------+------------------------------+               |                |             |
+| :ref:`ble_state`       | ``ble_peer_event``           |               |                |             |
++------------------------+------------------------------+               |                |             |
+| :ref:`battery_charger` | ``battery_state_event``      |               |                |             |
++------------------------+------------------------------+---------------+----------------+-------------+
+
+Configuration
+*************
+
+Module is enabled when ``CONFIG_DESKTOP_LED_ENABLE`` option is set (when :ref:`leds` is configured and enabled).
+For every board enabling this option you must define module configuration (in ``led_state_def.h`` file located in board-specific directory in the application configuration folder).
+
+Configuration consists of:
+
+* ``led_map`` - mapping from :cpp:enum:`led_id` values to IDs used by :ref:`leds` (if no physical LED is assigned to given :cpp:enum:`led_id` value, you must assign :c:macro:`LED_UNAVAILABLE` as ID used by :ref:`leds`),
+* ``led_system_state_effect`` - defines LED effects used to show system states (effect must be defined for every system state),
+* ``led_peer_state_effect`` - defines LED effects used to show Bluetooth peer states (effect must be defined for every state of every peer).
+
+LED effects are defined in ``led_effect.h`` file in common configuration folder (as macros).
```

Comment:
```suggestion
The LED effects are defined in the ``led_effect.h`` file in the common configuration folder (similarly as macros).
```
`(as macros)` can mean that the LED effects are defined as macros or that they are defined in a way similar to macros. Avoid this kind of ambiguities.

Reply:
I meant that LED effects are defined as macros, for example: `LED_EFFECT_ON()`.
After discussion with Paweł we decided, we will link the documentation from header file here to show all the macros (I will add this in separate PR).